app/gui/language/span-tree/src/generate.rs

```diff
@@ -528,6 +528,7 @@ fn generate_node_for_prefix_chain<T: Payload>(
     context: &impl Context,
 ) -> FallibleResult<Node<T>> {
     let app_base = ApplicationBase::from_prefix_chain(this);
+    let fallback_call_id = app_base.call_id;
```

Comment:
hmm, what is "fallback call id" ? Is this explained anywhere? If not, a short explanation here would be very helpful.